x/auth/ante/ante_test.go

```diff
@@ -597,11 +600,16 @@ func generatePubKeysAndSignatures(n int, msg []byte, _ bool) (pubkeys []crypto.P
 	signatures = make([][]byte, n)
 	for i := 0; i < n; i++ {
 		var privkey crypto.PrivKey
-		if rand.Int63()%2 == 0 {
-			privkey = ed25519.GenPrivKey()
-		} else {
-			privkey = secp256k1.GenPrivKey()
-		}
+		privkey = secp256k1.GenPrivKey()
+
+		// TODO: also generate ed25519 keys as below when ed25519 keys are actually supported,
```

Comment:
create issue?

Reply:
Sure, will create an issue and link

Reply:
@aaronc is this the relevant issue here or should I create new one https://github.com/cosmos/cosmos-sdk/issues/5694

Reply:
I think #4789 more fully address it @sahith-narahari